IdentityCore/src/webview/embeddedWebview/ui/mac/MSIDNTLMUIPrompt.m

```diff
@@ -46,7 +47,7 @@ + (void)dismissPrompt
     }];
 }
 
-+ (void)presentPrompt:(void (^)(NSString *username, NSString *password, BOOL cancel))completionHandler
++ (void)presentPromptWithWebView:(WKWebView *)webview completion:(void (^)(NSString *username, NSString *password, BOOL cancel))completionHandler
```

Comment:
Should we pass NSWindow instead of web view?

Reply:
This API is cross platform right now... and window doesn't really exist on iOS. We can make this API macOS specific and ifdef place where we call it. Passing webview is more inline with other challenge handlers where we just pass webview regardless if it's used or not. 

Let me know if you have preferences here.

Reply:
I see. Let's keep webview then.